### PR TITLE
Add URL as hyperlink in collection

### PR DESCRIPTION
**Pull request by rtritto**
This change is [Reviewable](https://reviewable.io/reviews/mongo-express/mongo-express/943)
- - -

Changes:
- Change _cursor_ to _auto_ in table of document listing page

Replace #296

### CODE REVIEW

**Comment by BlackthornYugen** [comment]
@rtritto before making a replacement branch can you check if the original author gave maintainers push permission to their branch?

File: lib/views/collection.html [hunk 1]
```diff
@@ -271,12 +271,19 @@
...
+                  <span class="glyphicon glyphicon-link"></span>
+                </button>
                 <button type="submit" class="btn btn-danger">
                 <span class="glyphicon glyphicon-trash"></span>
                 </button>
                 </form>
               {% endif %}
-            {{ document[column] | stringDocIDs | to_display | safe }}
+              {% if document[column]|is_URL %}
+                <a href="{{ document[column] }}" target="_blank">{{ document[column] }}</a>
```
[the remark this answers is not in the document]
Comment (shakaran) [reply lib/views/collection.html:283]: We should add here "noopener noreferrer" for the external links

**Comment by rtritto** [comment]
This fix also adds this new link button:

image

A document can be opened clicking on link button or clicking on the row.
Due to redundant behavior, can the link button be removed?

FYI @BlackthornYugen @shakaran

**Comment by OOCAZ** [comment]
Probably good to leave both, that way users have many options and makes it overly abundant to any user how to manage the users.

**Comment by shakaran** [comment]
Merging with both options. If we change the material UI later or improve the UI, we can rethink this

**Pull request merged**